apps/alerting/rules/pkg/apis/alerting/v0alpha1/ext.go

```diff
@@ -19,13 +19,17 @@ const (
 	FolderLabelKey      = FolderAnnotationKey
 )
 
+// NOTE: This is a copy of the constants from the alertrule package to avoid circular imports.
+// Keep in sync with pkg/services/ngalert/models/provisioning.go
 const (
-	ProvenanceStatusNone = ""
-	ProvenanceStatusAPI  = "api"
+	ProvenanceStatusNone                = ""
+	ProvenanceStatusAPI                 = "api"
+	ProvenanceStatusFile                = "file"
+	ProvenanceStatusConvertedPrometheus = "converted_prometheus"
```

Comment:
Would it be possible to move these constants from both places to somewhere else (to fix the circular dependency) and reuse, to avoid errors like this?

Reply:
I think it's possible, but I don't think it changes often enough to warrant the effort of creating the module and figuring out where to put it so that folks know where to find it. If it happens again I'd be very surprised, but perhaps we can have a test that validates that both of these sets are identical?